services/sources/publisher_backlist_service.py

```diff
@@ -121,9 +131,22 @@ def get_records(
             for records_value in json_dict['records']:
                 try:
                     record_metadata_dict = records_value['fields']
+                    file_id = f'{self.drive_service.id_from_url(record_metadata_dict["DRB_File Location"])}'
+                    file_name = self.drive_service.get_file_metadata(file_id).get("name")
+                    file = self.drive_service.get_drive_file(file_id)
+                    if not file:
+                        logger.warn(f'Could not retrieve file for {record_metadata_dict["id"]} from Drive, skipping')
+                        continue
+                    bucket = self.file_bucket # TODO: if record is limited access, upload to limited access bucket
+                    s3_path = f'{self.title_prefix}/{record_metadata_dict["Publisher (from Projects)"][0]}/{file_name}'
+                    s3_response = self.s3_manager.putObjectInBucket(file.getvalue(), s3_path, bucket)
+                    if not s3_response.get('ResponseMetadata').get('HTTPStatusCode') == 200:
+                        logger.warn(f'Could not upload file for {record_metadata_dict["id"]} to s3, skipping')
+                        continue
+                    s3_url = f'https://{bucket}.s3.amazonaws.com/{s3_path}'
```

Comment:
We could build this dynamically from the bucket host if we don't want to hard-code s3 here